src/cloudant/__init__.py

```diff
@@ -62,6 +63,57 @@ def cloudant(user, passwd, **kwargs):
     cloudant_session.disconnect()
 
 @contextlib.contextmanager
+def cloudant_bluemix(bm_service_name=None, **kwargs):
+    """
+    Provides a context manager to create a Cloudant session and provide access
+    to databases, docs etc.
+
+    :param str bm_service_name: Optional Bluemix service instance name. Only
+        required if multiple Cloudant services are available.
+    :param str encoder: Optional json Encoder object used to encode
+        documents for storage. Defaults to json.JSONEncoder.
+
+    Loads all configuration from the VCAP_SERVICES Cloud Foundry environment
+    variable. The VCAP_SERVICES variable contains connection information to
+    access a service instance. For example:
+
+    .. code-block:: none
+
+        {
+            "VCAP_SERVICES": {
+                "cloudantNoSQLDB": [
+                    {
+                        "credentials": {
+                            "username": "example",
+                            "password": "xxxxxxx",
+                            "host": "example.cloudant.com",
+                            "port": 443,
+                            "url": "https://example:xxxxxxx@example.cloudant.com"
+                        },
+                        "syslog_drain_url": null,
+                        "label": "cloudantNoSQLDB",
+                        "provider": null,
+                        "plan": "Lite",
+                        "name": "Cloudant NoSQL DB"
+                    }
+                ]
+        }
```

Comment:
This docstring code block appears to have a missing close parenthesis.  If you correct the structure of the block you can set it to `.. code-block:: json` once it is valid.  I think then it would render nicely in Sphinx.